test/libgit2.jl

```diff
@@ -346,6 +348,7 @@ mktempdir() do dir
                 LibGit2.add!(repo, test_file)
                 status = LibGit2.GitStatus(repo)
                 @test length(status) != 0
+                @test status[0] == nothing
```

Comment:
Correct me if I'm wrong but I think it means there are no changes to any tracked files.